### ISSUE TITLE
Query Filter: cannot shift+tab away when filter is focused

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wporg-mu-plugins#471]
Steps to reproduce:

1. Tab through the page until you reach a Query Filter component
2. Activate the filter
3. Close the filter
4. Press shift+tab to move backwards through the tab order

Expected result: focus moves to the previous focusable element

Actual result: focus stays on the Query Filter
